src/lib.rs

```diff
@@ -134,7 +133,7 @@ struct XprvData {
 }
 
 /// The `D` digest type param must implement SHA512. Use `sha2::Sha512` if in doubt.
-#[derive(Clone, Debug)]
+#[derive(Zeroize, ZeroizeOnDrop, Clone, Debug)]
```

Comment:
What's the purpose of this? Since XprvData already has these, and it's the only field in this struct, it seems that this does not add anything.

Reply:
Valid question, but XPriv does not zeroization out of the box which is not an issue on its own as you noted. However, if a downstream project wants to wrap `Xprv` and then have the zeroization derived on the wrapper it won't work because the wrapping type does not support zeroization. Hope that makes sense.

Reply:
Thanks, makes sense.